### DISCUSSION TITLE
API access to unpublished Chemotion ELN entries

### DISCUSSION THREAD

**Discussion by haeussma** [ComPlat/chemotion_REPO#102] [Q&A]
I've seen, that you provide a REST-API for entries of the Chemotion Repository. Is there also a way to access individual entries of a Chemotion ELN?

**Comment by cllde8** [comment]
Hi @haeussma ,

If you have installed the Chemotion ELN, you can access the Swagger documentation via `/swagger`.

Best regards,
Claire
